base/applications/drwtsn32/drwtsn32.rc

```diff
@@ -28,9 +28,12 @@
 #ifdef LANGUAGE_FR_FR
     #include "lang/fr-FR.rc"
 #endif
+#ifdef LANGUAGE_IT_IT
+    #include "lang/it-IT.rc"
+#endif
 #ifdef LANGUAGE_PL_PL
     #include "lang/pl-PL.rc"
 #endif
-#ifdef LANGUAGE_ZH_CN
```

Comment:
Huuhhhhhhhhhhhh why??
There is chinese translation too!!

Reply:
Because the Pull Request which has been merged for Chinese translation of Dr.Watson conflicted with mine. I had no choice but overriding the define macro as you can see.

I guess the reason why my PR was in conflict is because the branch which has been created from ```master``` is sooo old. Not sure though.

Reply:
Could you please help me on how can I fix it (and what should I do in the future when my Pull Request is in conflict?).